iron-form.html

```diff
@@ -122,9 +146,9 @@
 
     ready: function() {
       // Object that handles the ajax form submission request.
-      this._requestBot = document.createElement('iron-ajax');
-      this._requestBot.addEventListener('response', this._handleFormResponse.bind(this));
-      this._requestBot.addEventListener('error', this._handleFormError.bind(this));
+      this.request = document.createElement('iron-ajax');
```

Comment:
omg i thought this was pointing out a typo and I was like "I CAN'T SEE IT"

It's been a day.